extensions/notion/src/utils/notion/authorize.ts

```diff
@@ -0,0 +1,80 @@
+import { Client } from "@notionhq/client";
```

Comment:
You need to import fetch from `node-fetch` over here otherwise the OAuth login simply won't work 😬

Reply:
Oh, shoot! Will do.